lnrpc/rpc.proto

```diff
@@ -1426,6 +1426,8 @@ message ChannelCloseSummary {
         BREACH_CLOSE = 3;
         FUNDING_CANCELED = 4;
         ABANDONED = 5;
+        LOCAL_COOPERATIVE_CLOSE = 6;
```

Comment:
same here, make `COOPERATIVE_CLOSE` mean local close? (need to consider whether renaming it could break some users)

Reply:
Wouldn't that be incorrect for channels closed before this change? 
A channel marked `COOPERATIVE_CLOSE`, which we have repurposed to mean locally cooperatively closed, could actually have been closed by the remote party if was closed before this change. It makes it a bit ambiguous.

Reply:
IMO that is okay, since we don't have more information about who closed it anyway. But I could see it being confusing, as suddenly it looks like all your old closes were locally initiated. Do you think it would be a deal breaker?

Just sounds like a maintenance headache to keep three possible coop close types: `local`, `remote` and `unknown`.

Reply:
I'd vote for keeping cooperative/unknown, local, remote to not confuse users.

Reply:
I think it's a bit of a deal breaker tbh, want to reduce ambiguity not add a newer, more subtle form of it :(

Reply:
Could we go about this another way: instead of overloading the close types, what about adding a new field `close_initiator` or similar, that would augment the coop close information. That way we wouldn't have change the meaning of the existing enum.

My main worry about the current approach is that we are deprecating a DB field, and without a migration we will have to keep maintaining  all codepaths where it is used.

Reply:
So something that is exposed to the user along the lines of: 
`close_initiator` : `LOCAL`/`REMOTE` or `UNKNOWN` for older channels?

Sounds like a nice compromise to me, but we would need to migrate existing closes summaries.

Reply:
Yeah, something like that. Why would you need to migrate? We could make it a field that by default is `Unknown` and store the initiator under a new key for new channels.

Reply:
Discussed offline, close summaries store chanID keys rather than buckets, and have a bunch of optional fields at the end which makes quick addition of a new field tricky. We could add another bucket to `closed-chan-bucket` which has `chanID` buckets where new information can be stored. 

Since this is just a rpc change, opted to just get channel status from the historical channel bucket. If we want to fix up this bucket, it would be worth doing a full migration and getting rid of the optional fields as well, which is out of scope.